app/javascript/packs/game.js

```diff
@@ -0,0 +1,89 @@
+/**
```

Comment:
Import the shuffle function from question.js in order to use it in this file.

```suggestion
import { shuffle } from './model/question';
```

Reply:
Imported the function as suggested. Thanks :) 
Ref: 3714a4b